src/app/components/Profile/Profile.module.css

```diff
@@ -7,3 +7,31 @@
   -webkit-box-shadow: 0 0.125rem 0.3125rem 0 rgba(0, 0, 0, 0.25);
   box-shadow: 0 0.125rem 0.3125rem 0 rgba(0, 0, 0, 0.25);
 }
+
+.ProfileItem {
+  display: flex;
+  align-items: center;
+}
+
+.ProfileItemNotPicture {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  width: 100%;
+}
```

Comment:
You could use BEM principles here :) but in any case I would `camelCase` it not `PascalCase `

Reply:
Fixed in #20